### PR TITLE
Feature new modal mod

### PR DESCRIPTION
**Pull request by pochretien**

### CODE REVIEW

File: docs/_components/components/05_modal.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
 ---
 <!-- Modal triggers -->
 <button class="btn js-modal-trigger" data-modal="BaseModal">Basic modal</button>
+<button class="btn js-modal-trigger" data-modal="modSmall">Mod small</button>
```
Comment (JPLachance) [inline docs/_components/components/05_modal.html]: ModSmall instead of modSmall plz.

File: scss/components/modal.scss [hunk 2]
```diff
@@ -160,6 +165,10 @@ $base-modal-z-index: 1050;
     }
   }
 
+  &.mod-width-small > .modal-content {
+    width: $input-field-max-width + 2 * $header-padding;
```
Comment (JPLachance) [inline scss/components/modal.scss]: Not so sure about this one. InputField should have a mod-full-width and the modal should have his own width. I'll need the mod-full-width for inputs really soon, so simply add it.

**Comment by JPLachance** [comment]
Fix it then :shipit: !

**Comment by wfortin** [comment]
Nothing to add. 🚢  it

File: docs/_components/components/05_modal.html [hunk 3]
```diff
@@ -31,6 +32,27 @@
...
     </div>
 </div>
+<div class="modal-container mod-small" id="modSmall">
+    <div class="modal-content">
+        <header class="modal-header">
+            <h1>Mod small</h1>
+            <span class="small-close js-modal-close">
+                <svg width="512" height="512" viewBox="0 0 512 512" class="icon fill-pure-white">
+                    <path d="M480.002 96l-64-63.998-160 159.998-160-160-64 63.998L192.004 256l-160 159.998 64 64 160-159.998 159.994 159.994 64-64L320.003 256"></path>
+                </svg>
```
Comment (GermainBergeron) [inline docs/_components/components/05_modal.html]: You can replace the svg with `{{ site.data.icons['your-svg-name'] }}`

**Pull request merged**